### ISSUE TITLE
reicast fails to load .gdi

### ISSUE THREAD

**Issue by crocket** [reicast/reicast-emulator#1146]
version : a50861b249b83abbdc2a9281434b75b25520baa5
OS : Gentoo Linux
[reicast.log](https://github.com/reicast/reicast-emulator/files/1978267/reicast.log)

![2018-05-07_mon_01 46 25](https://user-images.githubusercontent.com/748856/39681276-211ddbfc-51e4-11e8-8714-b636ede6f9b4.png)
![2018-05-07_mon_01 46 34](https://user-images.githubusercontent.com/748856/39681277-21478484-51e4-11e8-8c9a-cefdbb17e231.png)
![2018-05-07_mon_01 46 37](https://user-images.githubusercontent.com/748856/39681278-2184b82c-51e4-11e8-92dd-b8248dbf7bfd.png)

**Comment by crocket** [comment]
I don't know about the region, but the same bios files work well with reicast-libretro.

**Comment by LoungeKatt** [comment]
Other variations of the emulator may not "respect" the region of the BIOS, but neither configuration is wrong. Unfortunately, whether or not it works with reicast-libretro doesn't mean much if that is the issue.

**Comment by crocket** [comment]
How can I check the region? Where is the region stored? The ROMs? The BIOS files?

**Comment by LoungeKatt** [comment]
It would be the region where you purchased the Dreamcast or the region listed on the disk / jacket of the game. Keep in mind, we have to assume you made a personal backup of your BIOS and games. That is the only legal way to play to them with an emulator (and the only thing we can support). If you got them from the internet, then you will most likely find your answers by going back to that site.

**Comment by crocket** [comment]
On the commit b4631f6715ab7ee8483ebcef37a97f6e3194602e, this issue didn't exist.
But, b4631f6715ab7ee8483ebcef37a97f6e3194602e had a problem with gdi path parsing which was supposed to be fixed with https://github.com/reicast/reicast-emulator/pull/1098

**Comment by baka0815** [comment]
This should be fixed by #1191 too.
The output in the log looks identical:
````
GDI : 3 tracks
file[1] "Jet Grind Radio (USA) (Track 1).bin": FAD:0, CTRL:4, SSIZE:2352, OFFSET:0
file[1] "": FAD:0, CTRL:4, SSIZE:2352, OFFSET:0
file[1] "": FAD:0, CTRL:4, SSIZE:2352, OFFSET:0
gdrom: Opened image "Jet Grind Radio (USA).gdi"
````
